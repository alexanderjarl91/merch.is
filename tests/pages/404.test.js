import { shallow } from "enzyme";
import Custom404 from "../../pages/404";

it("expect Custom404 component to match Snapshot", () => {
  expect(shallow(<Custom404 />)).toMatchSnapshot();
});

it("expect to render component Custom404", () => {
  expect(shallow(<Custom404 />).length).toEqual(1);
});
