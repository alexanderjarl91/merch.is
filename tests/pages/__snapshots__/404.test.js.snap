// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect Custom404 component to match Snapshot 1`] = `ShallowWrapper {}`;
